product_docs/docs/postgres_distributed_for_kubernetes/1/identify_images/identify_image_name.mdx

```diff
@@ -1,56 +1,52 @@
+The operand image name is composed of the Postgres type, EDB Postgres Distributed, their version numbers, and the build version.
 
-**The Postgres type:** The first step towards identifying your image names is knowing which Postgres type you are going to use, and how it translates to an image name. 
+**Postgres type:** The first part of the image name is the Postgres type you're using. The table shows how Postgres types correspond to this part of the image name.
 
 | Postgres type                | Image name            |
 |------------------------------|-----------------------|
 | EDB Postgres Advanced Server | edb-postgres-advanced |
 | EDB Postgres Extended Server | edb-postgres-extended |
 | PostgreSQL                   | postgresql            |
 
-**EDB Postgres Distributed:** The next step is identifying how Postgres Distributed translates to an image name. This is the same for all environment configurations: 
+**EDB Postgres Distributed:** For all environment configurations, `pgd` is the next part of the image name.
 
-| Product name             | Image name |
-|--------------------------|------------|
-| EDB Postgres Distributed | pgd        |
+**Version numbers:** The versions of the Postgres type and PGD, separated by a dash, are next in the image name.
 
-**Build version:** Last, the build version is 1, unless a new image is created in the scope of a security patch.
+**Build version:** The build version is 1, unless a new image is created in the scope of a security patch.
 
 ### Assembling the operand image name 
 
-These identifiers, together with their versions, translate to an operand image name. Here are a few examples: 
+These identifiers, together with their versions, create the operand image name. The table shows some examples. 
 
 | Postgres version           | EDB Postgres Distributed version | Build | Operand image name                       |
 |----------------------------|----------------------------------|-------|------------------------------------------|
 | EDB Postgres Advanced 16.2 | PGD 5.4.1                        | 1     | edb-postgres-advanced-pgd:16.2.0-5.4.1-1 |
 | EDB Postgres Extended 15.6 | PGD 5.4.0                        | 1     | edb-postgres-extended-pgd:15.6-5.4.0-1   |
 | PostgreSQL 16.2            | PGD 5.4.1                        | 1     | postgresql-pgd:16.2-5.4.1-1              |
 
-For an overview of Postgres compatibility, see [Platform compatibility](https://www.enterprisedb.com/resources/platform-compatibility).
-
 !!! Note Postgres version format
-    For PostgreSQL and EDB Extended Server images, the Postgres version is provided in `x.y` format, for example, 15.6. For EDB Postgres Advanced Server, the Postgres version is provided in `x.y.z` format, for example, 15.6.2.
+    For PostgreSQL and EDB Extended Server images, the Postgres version is in `x.y` format, for example, 15.6. For EDB Postgres Advanced Server, the Postgres version is in `x.y.z` format, for example, 15.6.2.
```

Comment:
Passive voice was confusing here (is provided in). I wasn't sure if this was a user action. But in re-reading the content, I think this is about selecting the image name, not creating one.

Reply:
You are correct; this is more about finding a different image name if you don't want to install what is provided by default (for example, because you are using an older Postgres version). I think your suggestion works well here.